### PR TITLE
afew: correct dependencies and install manpage

### PR DESCRIPTION
**Pull request by sgn**

### CODE REVIEW

File: srcpkgs/afew/template [hunk 1]
```diff
@@ -1,25 +1,25 @@
 # Template file for 'afew'
 pkgname=afew
-version=1.3.0
+version=2.0.0
 revision=1
 archs=noarch
 build_style=python3-module
 pycompile_module="afew"
-hostmakedepends="git python3-setuptools python3-Sphinx pkg-config"
-depends="python3-setuptools notmuch-python3 python3-dkimpy"
```
Comment (Chocimier) [inline srcpkgs/afew/template]: python3-setuptools is needed, and also python3-chardet

File: srcpkgs/afew/template [hunk 2]
```diff
@@ -1,25 +1,25 @@
...
-short_desc="An initial tagging script for notmuch mail"
+hostmakedepends="python3-setuptools python3-Sphinx pkg-config"
+depends="notmuch-python3 python3-dkimpy"
+checkdepends="python3-pytest $depends"
```
Comment (Chocimier) [inline srcpkgs/afew/template]: Good you care for that!

File: srcpkgs/afew/template (hunk 1 above)
Comment (sgn) [reply srcpkgs/afew/template]: python3-chardet is required but I don't think afew requires python3-setuptools.

I stopped using afew for a while, though

Comment (sgn) [reply srcpkgs/afew/template]: It seems like afew doesn't require setuptools directly,
afew requires chardet and chardet requires setuptools.

**Pull request merged**